source/reference/about-mongosync.txt

```diff
@@ -1,50 +1,43 @@
 .. _about-mongosync:
 
-========================
+====================
 About ``mongosync``
-========================
+====================
 
 The ``mongosync`` binary is the primary process used in {+c2c-product-name+}. 
 ``mongosync`` migrates data from one cluster to another and keeps the 
 clusters in continuous sync. You can use ``mongosync`` to create dedicated 
 analytics, development, or testing clusters that mirror your production 
 environment. 
 
-``mongosync`` enters different states depending on the requests it receives. 
-The current ``mongosync`` state determines which API operations you can run. 
+``mongosync`` keeps track of its current actions through states. ``mongosync`` 
+enters different states depending on the requests it receives. The current 
+``mongosync`` state determines which API operations you can run. 
```

Comment:
According to the internal docs, "States represent Mongosync’s current action. This is a public data point." I don't originally have this in the docs, but users can track monosync's progress by calling the progress API endpoint, which will tell them what state they are currently in. So I guess it is both an actual process and a tracking mechanism for the user. Do you think I should talk about the progress endpoint on this page to clarify? I originally thought it was unnecessary because I didn't want to give the user instructions.